pandas/tests/io/json/test_normalize.py

```diff
@@ -5,10 +5,9 @@
 
 from pandas.compat import PY36
 
-from pandas import DataFrame, Index
+from pandas import DataFrame, Index, json_normalize
```

Comment:
Can you add a test here that importing from pandas.io.json now throws a `FutureWarning`?

Reply:
Sure, I will do that.